.github/workflows/pre-merge.yaml

```
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        jdk: [8, 11]
+        jdk: [8, 11, 15, 16]
```

```suggestion
        jdk: [8, 11, 16]
```
We generally support only LTS and the latest released version

---

I've updated this as well